README.md

```diff
@@ -1,7 +1,152 @@
-# src-mq 
+# src-mq
 
-> Breakpoint media queries in JS.
+🤓 Legible breakpoints for JavaScript.
 
-[![Build Status](https://travis-ci.org/src-mq/src-mq.svg?branch=master)](https://travis-ci.org/src-mq/src-mq) 
+[![Build Status](https://travis-ci.org/src-mq/src-mq.svg?branch=master)](https://travis-ci.org/src-mq/src-mq)
 [![codecov](https://codecov.io/gh/src-mq/src-mq/branch/master/graph/badge.svg)](https://codecov.io/gh/src-mq/src-mq)
 
+## Defaults
+
+### Breakpoints
+
+- `xxSmall` (320 pixels)
+- `xSmall` (375 pixels)
+- `small` (480 pixels)
+- `medium` (740 pixels)
+- `large` (980 pixels)
+- `xLarge` (1140 pixels)
+- `xxLarge` (1300 pixels)
+
+_n.b. breakpoint values are converted to ems, assuming an em is 16 pixels._
+
+### @media type
+The default is `all`.
+
+## Usage
+
+There are two ways of applying breakpoints:
+
+- `from`
+- `until`
+
+```js
+import { from, until } from 'src-mq'
+
+const styles = {
+
+	[until.medium]: {
+		color: 'green',
+	},
+
+	[from.large]: {
+		color: 'red',
+	},
+
+}
+```
+
+_n.b. `from`  applies from the breakpoint, `until` applies until the breakpoint minus one pixel._
+
+### Ranges
+
+`from` can also chain an `until`, to apply styles _between_ two breakpoints:
+
+```js
+import { from } from 'src-mq'
+
+const styles = {
+
+	[from.small.until.medium]: {
+		color: 'blue',
+	},
+
+}
+```
+
+### Custom @media types
+
+Both techniques can chain an optional `for()` method, allowing you set a media type per query:
+
+```js
+import { from, until } from 'src-mq'
+
+const styles = {
+
+	[until.medium.for('print')]: {
+		color: 'green',
+	},
+
+	[from.large.for('screen and print')]: {
```

Comment:
This should probably be `for('screen, print`)`

Comma is a [logical `or` operator](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#(comma))

Reply:
👍